support.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Support Resources
+summary: Find support resources if you encounter a problem using TiDB.
+---
+
+# Support Resources
+
+If you encounter a problem when you use TiDB, you can reach out for support from PingCAP or the TiDB community via the following methods:
+
++ Get support from PingCAP (requires subscription to TiDB Enterprise Edition):
+
+    - [Submit a request](https://support.pingcap.com/hc/en-us)
```

Comment:
How about using a shorter one here?
```suggestion
    - [Submit a request](https://support.pingcap.com/)
```

Reply:
As I tested it, even with a global VPN on, `https://support.pingcap.com/` directed me to the Chinese site. I'm sure if it's only me...